core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/add-hint-modal.controller.spec.ts

```diff
@@ -62,13 +62,14 @@ describe('Add Hint Modal Controller', function() {
     });
   }));
 
-  it('should evaluate scope variable values correctly', function() {
-    expect($scope.tmpHint).toBe('');
-    expect($scope.addHintForm).toEqual({});
-    expect($scope.hintIndex).toBe(5);
-  });
+  it('should initialize $scope properties after controller is initialized',
+    function() {
+      expect($scope.tmpHint).toBe('');
+      expect($scope.addHintForm).toEqual({});
+      expect($scope.hintIndex).toBe(5);
+    });
 
-  it('should save hint successfully', function() {
+  it('should close modal when saving hint', function() {
```

Comment:
Done.

Reply:
Hm, I'm confused. I don't think we should change it because I'm asserting when `$uibModal.close` is called, not when the hint is saved. Here and elsewhere.

Reply:
That's from the perspective of the code. From the perspective of a user, he clicks the button to close and then expects that the hint is saved. That's why I was suggesting renaming it as above.

Reply:
Sure, I fixed it.